GitHubApp/ReadShortText.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ReadShortText.swift
+//  GitHubApp
+//
+//  Created by Van Lee Chigwada on 2024/09/16.
+//
+
+import SwiftUI
+
+struct ReadShortText: View {
+    
+    var body: some View {
+        
+        Text("Some short text to be read")
```

Comment:
You should not hard code text in your UI, create a variable so it can be changed.